src/utils/get-tag-links-from-meta.js

```diff
@@ -0,0 +1,20 @@
+export const getTagLinksFromMeta = meta => {
+    const mappedTags = mapTagTypeToUrl(meta.tags, 'tag');
+    const mappedLanguageTags = mapTagTypeToUrl(meta.languages, 'language');
+    const mappedProductTags = mapTagTypeToUrl(meta.products, 'product');
+    const allTags = [
```

Comment:
I didn't think it was necessary now to worry about uniqueness between all the various types of tags.